Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneEventCollector/TrendMicroVisionOneEventCollector_test.py

```diff
@@ -31,14 +31,35 @@ def get_url_params(url: str) -> Dict[str, str]:
     }
 
 
-def create_any_type_logs(start: int, end: int, created_time_field: str, id_field_name: str, extra_seconds: int = 0):
+def create_any_type_logs(
+    start: int,
+    end: int,
+    created_time_field: str,
+    id_field_name: str,
+    last_event_fetch_time: str | None,
+    extra_seconds: int = 0,
+    ascending_order: bool = False,
+):
+    last_event_fetch_time = last_event_fetch_time or (datetime.now() - timedelta(minutes=1)).strftime(DATE_FORMAT)
+    if ascending_order:
+        return [
+            {
+                id_field_name: i,
+                created_time_field: (  # type: ignore
+                    dateparser.parse(last_event_fetch_time) + timedelta(seconds=i + extra_seconds)  # type: ignore
+                ).strftime(DATE_FORMAT) if created_time_field != 'eventTime' else int(
+                    (dateparser.parse(last_event_fetch_time) + timedelta(seconds=i + extra_seconds)).timestamp()  # type: ignore
+                ) * 1000
+            } for i in range(start + 1, end + 1)
+        ]
+    # descending order
     return [
         {
-            id_field_name: i,
+            id_field_name: end - i + 1,
```

Comment:
whats this line mean?

Reply:
its just a way to represent the IDs in a more convenient way when the events are returned in descending order, ill document it, also used for testing